glommio/src/task/header.rs

```diff
@@ -82,12 +89,38 @@ impl Header {
         // Put the waker into the awaiter field.
         abort_on_panic(|| self.awaiter = Some(waker.clone()));
     }
+
+    #[allow(dead_code)]
```

Comment:
isn't it better to make this conditional on `debugging`?

Reply:
Yes, that would be better.